### PR TITLE
Consumer subscribe Unit Test

### PR DESCRIPTION
**Pull request by h1alexbel**

### CODE REVIEW

**Comment by h1alexbel** [comment]
@rultor merge

File: src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java [hunk 1]
```diff
@@ -9,11 +16,22 @@
...
+      new KfConsumer<>(
+        new KfConsumerSettings<String, String>(
+          new XMLDocument(
+            new File("src/test/resources/consumer.xml")
+          )
+        ).consumer()
+      );
+    assertDoesNotThrow(
+      () ->
+        consumer.subscribe(
```
Comment (l3r8yJ) [inline src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java:31]: @h1alexbel maybe if the subscription was successful, we can return some kind of identifier?

Comment (h1alexbel) [reply src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java:31]: @l3r8yJ 
maybe, I still investigating the best fits in consumer methods: ```subscribe``` and ```dataized = fetch```
but internal implementation of ```subscribe``` returns ```void```

Comment (l3r8yJ) [reply src/test/java/io/github/eocqrs/kafka/consumer/KfConsumerTest.java:31]: @h1alexbel got it

**Pull request merged**